src/BasicRum/Beacon/Importer/Process/Beacon/RtTimingsNormalizer.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\BasicRum\Beacon\Importer\Process\Beacon;
+
+class RtTimingsNormalizer
+{
+    public function normalize(array $timing)
+    {
+        $entries = [
```

Comment:
We already defined an array with the keys we expect to import why don't we use similar technique with foreach loop in order to avoid the repeated code in the if conditions below?

Reference: https://github.com/basicrum/backoffice/blob/master/src/BasicRum/Beacon/Importer/Process/Beacon/NavigationTimingsNormalizer.php#L40